apps/web/src/pages/DirectivePage/DirectivePage.tsx

```diff
@@ -385,63 +376,306 @@ export const Component = () => {
+              })}
+            </Heading>
+            <div data-h2-margin="base(0, 0, x1, 0)">
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The Office of the Chief Information Officer of Canada (OCIO) has been closely monitoring the effectiveness of the Directive on Digital Talent since it came into effect in April 2023.",
+                  id: "BVgb+I",
+                  description:
+                    "first paragraph describing the 2024 changes to the directive on digital talent",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "As the Directive is performing strongly, it will remain as is.",
+                  id: "qQmKgl",
+                  description:
+                    "second paragraph describing the 2024 changes to the directive on digital talent",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "While the Mandatory Procedures on Digital Talent are also effective, there are opportunities for improved clarity, reduced reporting burden for departments, and better alignment with the new <link>Mandatory Procedures for Business Owners When Procuring Professional Services</link> led by the Office of the Comptroller General of Canada (OCG).",
+                    id: "HRmtdK",
+                    description:
+                      "third paragraph describing the 2024 changes to the directive on digital talent",
+                  },
+                  {
+                    link: (chunks: ReactNode) =>
+                      procurementLink(locale, chunks),
+                  },
+                )}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "As a result, OCIO is introducing some updates to the Mandatory Procedures on Digital Talent, effective September 30th, 2024.",
+                  id: "deiUXj",
+                  description:
+                    "fourth paragraph describing the 2024 changes to the directive on digital talent",
+                })}
+              </p>
+            </div>
+            <CardBasic>
+              <ul
+                data-h2-margin="base(0)"
+                data-h2-padding="base(0, 0, 0, x1)"
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-gap="base(x0.25)"
+              >
+                <li>
+                  <strong>
+                    {intl.formatMessage({
+                      defaultMessage: "Removal of the Forward Talent Plan",
+                      id: "O8YvBD",
+                      description: "first 2024 key change to the directive",
+                    })}
+                  </strong>
```

Comment:
I'm not sure how or why but our set default styling for `strong` is actually `100` weight lower than our typical (`600` vs `700`). 

I'm not even sure how to fix this because it seems to be coming from hydrogen itself since we are not setting it anywhere but it does appear in our CSS file.

![2024-09-18_09-57](https://github.com/user-attachments/assets/eabc04b4-ac33-4de3-a6df-5bd9f5d8ae0a)

Reply:
Hmm, yeah, it's not in the hydrogen config file.  Must be some sort of standardized base styles that hydrogen uses.  Would you prefer a font-weight:body instead?

Reply:
I think so to make it consistent with our other bold text. However I am not absolutely certain. It may be `600` for a reason :thinking:

Reply:
I've changed it here: [switch strongs to bold](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11499/commits/db24cb7e858224bde732008e1048e4ecfc28109f)